src/python/pants/backend/debian/target_types.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.core.goals.package import OutputPathField
+from pants.engine.target import (
+    COMMON_TARGET_FIELDS,
+    DictStringToStringField,
+    SpecialCasedDependencies,
+    StringField,
+    Target,
+)
+from pants.util.docutil import doc_url
+
+
+class DebianControlFile(StringField):
+    alias = "control"
+    help = "Path to a Debian control file for the package to be produced."
+
+
+class DebianSymlinks(DictStringToStringField):
+    alias = "symlinks"
+    help = "Symlinks to create for each target being packaged."
+
+
+class DebianInstallPrefix(StringField):
+    alias = "install_prefix"
+    default = "/opt"
+    help = "Absolute path to a directory where Debian package will be installed to."
+
+
+class DebianPackageDependencies(SpecialCasedDependencies):
+    alias = "packages"
+    help = (
```

Comment:
Should this have `required = True`? Is it valid to have a `debian_package` with no packages directly in it, only the control file?

Reply:
Oh yes, let's make it required, because a Debian package with no targets is pretty useless :)